bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -150,6 +150,17 @@ ldap_validate() {
     [[ "$error_code" -eq 0 ]] || exit "$error_code"
 }
 
+########################
+# Check if OpenLDAP is ready to answer
+# Arguments:
+#   None
+# Returns:
+#   Whether slapd is ready to answer
+#########################
+is_ldap_ready() {
+    is_ldap_running && debug_execute ldapsearch -Y EXTERNAL -H "ldapi:///" -b cn=config -s base || false
```

Comment:
shellcheck is detecting a [SC2015](https://github.com/koalaman/shellcheck/wiki/SC2015) here. The assertion is right, since the C part would be a false, so we must skip it with:

```
is_ldap_ready() {
    # shellcheck disable=SC2015
    is_ldap_running && debug_execute ldapsearch -Y EXTERNAL -H "ldapi:///" -b cn=config -s base || false
}
```

Reply:
Sorry for this, I completely forget to put the verify label to trigger the VIB verify job :(